src/Features/RedirectRedirectPages.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace P4\MasterTheme\Features;
+
+use P4\MasterTheme\Feature;
+
+/**
+ * @see description().
+ */
+class RedirectRedirectPages extends Feature {
+
+	/**
+	 * @inheritDoc
+	 */
+	public static function id(): string {
+		return 'redirect_redirect_pages';
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	protected static function name(): string {
+		return __( 'Redirect redirect pages', 'planet4-master-theme-backend' );
```

Comment:
Just to make it more explicit: _"Redirect unused Tag pages"_.

(and by "explicit" I mean that the title should be about "Tag pages")